certified-connectors/timeghost/apiDefinition.swagger.json

```diff
@@ -400,6 +399,192 @@
         ]
       }
     },
+    "/update/projects/{ProjectId}/tags": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "$ref": "#/definitions/ProjectsResponse"
+            }
+          },
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Update Tags of an Project",
+        "description": "Update project tags in a workspace.",
+        "operationId": "Update_Project_Tags",
+        "parameters": [
+          {
+            "name": "workspace-id",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "Get_Currentuser",
+              "value-path": "id",
+              "value-collection": "workspaces",
+              "value-title": "name"
+            },
+            "description": "Enter the workspace (workspace id) in which your data is stored",
+            "x-ms-summary": "Workspace"
+          },
+          {
+            "name": "ProjectId",
+            "in": "path",
+            "required": true,
+            "x-ms-url-encoding": "single",
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "Get_Projects",
+              "value-path": "id",
+              "value-title": "name",
+              "parameters": {
+                "workspace-id": {
+                  "parameter": "workspace-id"
+                }
+              }
+            },
+            "description": "Project (or Project ID)",
+            "x-ms-summary": "Project"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ProjectTagsUpdate"
+            }
+          }
+        ]
+      }
+    },
+    "/update/projects/{ProjectId}/users": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "$ref": "#/definitions/ProjectsResponse"
+            }
+          },
+          "default": {

```

Comment:
@Marc-Hoch Default responses or empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger.

Applied the same comment for all the similar values.